src/metrics/loc.rs

```diff
@@ -457,6 +468,109 @@ mod tests {
         );
     }
 
+    #[test]
+    fn python_negative_blank() {
+        check_metrics!(
+            "def ConnectToUpdateServer():
```

Comment:
Can you also add the other two test cases I mentioned in the earlier review?

Reply:
Sure! :)